src/comps/Deposit.tsx

```diff
@@ -80,14 +80,15 @@ const DepositFlowAmount = ({ setStep, setAmount }: DepositFlowAmountProps) => {
     amount: yup
       .number()
       // dust amount is in sats
-      .min(10_000, `Minimum deposit amount is 10,000 sats`)
-      .max(currentCap, `Current deposit cap is ${currentCap} sats`)
+      .min(10_000 / 1e8, `Minimum deposit amount is ${10_000 / 1e8} BTC`)
```

Comment:
Shouldn't this limit come from Emily rather than being hardened as 10_000 sats?

Reply:
does emily provide a lower limit?

Reply:
If it doesn't and you are adding it for UX, it should be at least 100_000 sats

Reply:
got it!